pyproject.toml

```diff
@@ -14,6 +14,199 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+
+[build-system]
+requires = ["scikit-build-core>=0.7.0"]
```

Comment:
I saw ffi requires the minimal version of 0.10.0.  Do we want to align the minimal version requirement?

https://github.com/apache/tvm/blob/472b2fcd22c4147c35cbdf884d232f69bd86a23b/ffi/pyproject.toml#L51-L53